Classes/Domain/Repository/ApacheSolrDocumentRepository.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace ApacheSolrForTypo3\Solr\Domain\Repository;
```

Comment:
I would propose to add a unit and integration test to make sure that we can retrieve the documents form solr in the future.